docs/elements/hx-file-input/index.html

```diff
@@ -0,0 +1,77 @@
+---
+title: <hx-file-input>
+minver: 0.13.0
+also:
+  components/files: Files
+  components/icons: Icons
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+  <section>
+    <p>
+      The custom <code>{{page.title}}</code> element is a button
+      representation of a file input.
```

Comment:
`The custom <hx-file-input> element renders a button to access the system file selector dialog.`